### ISSUE TITLE
upgrade example from plots to makie, polar plot

### ISSUE THREAD

**Issue by lazarusA** [JuliaDataCubes/YAXArrays.jl#276]
Maybe do a Makie version for [this](https://juliadatacubes.github.io/YAXArrays.jl/dev/examples/generated/UserGuide/examples_from_esdl_study_1/) so that we can avoid future conflicts in CI regarding conda and PyCall.

**Comment by meggart** [comment]
Yes, I discussed this with Lina as well. The main problem is that we need this merged: https://github.com/MakieOrg/Makie.jl/pull/2014
